### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by davidoesch**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+web-mapviewer is available for use under the following license, commonly known
+as the 3-clause (or "modified") BSD license:
+
+==============================
+
+Copyright (c) 2019-2020, swisstopo
```
Comment (pakb) [inline LICENSE.md:6]: I thought we would write 2020 only, but if we go the way of multiple years I would prefer to write `2019-present` so we don't have to go back each new year's eve to edit this year

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+==============================
+
+Portions of web-mapviewer are based on works by others. A non-extensive list:
+
+[VueJS Source Code](https://github.com/vuejs/vue/blob/dev/LICENSE)
+
```
Comment (pakb) [inline LICENSE.md:24]: I've also added bootstrap in the end, so we can keep this portion from the other project's licence too.

**Review by pakb — CHANGES_REQUESTED** [review]
You had the same idea it seems 😄 (https://github.com/geoadmin/web-mapviewer/pull/10)

File: LICENSE.md (hunk 1 above)
Comment (davidoesch) [reply LICENSE.md:6]: ok 2020 only

File: LICENSE.md (hunk 2 above)
Comment (davidoesch) [reply LICENSE.md:24]: ok

**Comment by davidoesch** [comment]
@boecklic  needs a review from you as well

**Review by boecklic — APPROVED** [review]

**Review by pakb — APPROVED** [review]

**Pull request merged**